### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by MichaelZaporozhets**
clean root directory

### CODE REVIEW

**Comment by MichaelZaporozhets** [comment]
> You’ve updated `package-lock.json`, was that intentional?

now it is

**Comment by MichaelZaporozhets** [comment]
@cla-bot check

File: bower.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "px-vis-timeseries",
-  "version": "4.0.1",
+  "version": "4.1.0",
```
Comment (evanjd) [inline bower.json]: Why is this a minor bump? No features have been added. 

As this changes nothing user or developer facing (ie. someone who has px-vis-timeseries as a project dependency), I don’t think a new version should be released.

Comment (MichaelZaporozhets) [reply bower.json]: @evanjd figured it wasn't a patch either. fair enough-

**Pull request closed without merging**